codebundles/istio-sidecar-check/runbook.robot

```diff
@@ -0,0 +1,129 @@
+*** Settings ***
+Documentation       Checks for Istio sidecar injection across deployments in all namespaces.
+Metadata           Author    Helpful Assistant
+Metadata           Display Name    Istio Sidecar Injection Check
+Metadata           Supports    Kubernetes,AKS,EKS,GKE,OpenShift
+
+Library             BuiltIn
+Library             RW.Core
+Library             RW.CLI
+Library             RW.platform
+Library             RW.K8sHelper
+Library             OperatingSystem
+Library             String
+
+Suite Setup         Suite Initialization
+
+*** Tasks ***
+
+Check Deployments For Istio Sidecar Injection
```

Comment:
This title needs more of a :What" and "Where" - Task titles are very important, as they are indexed by the system and suggested when some issue arises. Imagine if you have multiple clusters and multiple Istio instances to check, we would need a better "Where". 

For example, this might be "Validate Istio Sidecar Injection for Cluster [CLUSTER_NAME]"

Please review some of @saurabh3460 's examples for more ideas on this.